From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@TimMurphy__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Lisaco88 committed:
edit pass: site-recovery-replicate-vmware-to-azure

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into site-recovery-replicate-vmware-to-azure

Lisaco88 committed:
edit pass: site-recovery-replicate-vmware-to-azure

Lisaco88 committed:
Update site-recovery-replicate-vmware-to-azure.md

Commit:
added csharp fw update article and placeholder media files

Commit:
added screenshots

Commit:
added selector and TOC entries

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into csharp-csharp-FW-update-article
:sync with master

Commit:
fixed grammer in instructions

Commit:
got rid of extra LF in step 13 that was causing code section to break

Commit:
changed screenshots to 8 bit samples

itechedit committed:
edit pass: how-to-read-write-files

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into how-to-read-write-files

itechedit committed:
edit pass: how-to-read-write-files

itechedit committed:
Update how-to-read-write-files.md

itechedit committed:
Update how-to-read-write-files.md

itechedit committed:
Update how-to-read-write-files.md

paulathurman committed:
edit pass: data-prep-python-execution-api

ChrisGMsft committed:
Details on max/min values and sizes for twins

Added max/min values for integers, updated max string size, and clarified max sizes for device segments.

ChrisGMsft committed:
Update iot-hub-devguide-device-twins.md

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into data-prep-python-execution-api

paulathurman committed:
edit pass: data-prep-python-articles

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into data-prep-python-execution-api

paulathurman committed:
edit pass: data-prep-python-articles

paulathurman committed:
edit pass: data-prep-python-articles

iainfoulds committed:
Updating PowerShell autoscale

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into data-prep-python-execution-api

paulathurman committed:
edit pass: data-prep-python-articles

paulathurman committed:
edit pass: data-prep-python-article

Commit:
incorporated dev review

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into csharp-csharp-FW-update-article
Sync with master

iainfoulds committed:
Revisions. Refactoring CLI auto scale doc

iainfoulds committed:
Cleaning CLI doc

iainfoulds committed:
Further revisions on PowerShell + CLI docs. Ready for review.

iainfoulds committed:
Acrolinx updates

iainfoulds committed:
Final revisions

Commit:
riskevents01

Commit:
riskevents01

iainfoulds committed:
Adding portal draft and screenshots

iainfoulds committed:
Adding CLI scheduled auto scale rules. Portal auto scale schedule rules

iainfoulds committed:
Revisions. Updating TOC. Adding redirects for doc renames

Commit:
incorporated Dominic's feedback

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into csharp-csharp-FW-update-article
sync with master

iainfoulds committed:
Resolving merge conflict in redirect file

kfile committed:
Updated date

kfile committed:
Corrected date

wmgries committed:
Fix minor typo in the break out for storage services.

wmgries committed:
Remove Azure File Sync *service* specific stuff per @tysonn.

iainfoulds committed:
Refactoring overview doc

mahesh-unnikrishnan committed:
Update active-directory-ds-join-windows-vm-template.md

mahesh-unnikrishnan committed:
Update active-directory-ds-join-windows-vm-template.md

mahesh-unnikrishnan committed:
Update active-directory-ds-join-windows-vm-template.md

mahesh-unnikrishnan committed:
Update active-directory-ds-join-windows-vm-template.md

mahesh-unnikrishnan committed:
Delete active-directory-ds-admin-guide-join-windows-vm-classic-powershell.md

mahesh-unnikrishnan committed:
Update TOC.md

mahesh-unnikrishnan committed:
Update .openpublishing.redirection.json

Commit:
added fenced code blocks with language tags

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into csharp-csharp-FW-update-article
sync with master

sethmanheim committed:
Adding article on Event Hubs monitoring

Commit:
removed indents from some fenced blocks to test

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into csharp-csharp-FW-update-article
sync with master

sethmanheim committed:
Fix metadata

sethmanheim committed:
Tweaks

sethmanheim committed:
Fixing link

iainfoulds committed:
Updating terminology and overview intro

sethmanheim committed:
Adding to TOC

Commit:
try 4 char indent in numbered code blocks

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into csharp-csharp-FW-update-article
sync with master

iainfoulds committed:
Updating core VMSS overview doc

iainfoulds committed:
Acrolinx update

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into data-prep-python-execution-api

sethmanheim committed:
Fixing tables and links

Commit:
final shot at code indents?

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into csharp-csharp-FW-update-article
sync with master

sethmanheim committed:
Adding link to GitHub sample

paulathurman committed:
edit pass: data-prep-python-article

carolinacmoravia committed:
C5338: Broken format link in target versions due to missing breaklines

Hello, @tamram,
There are missing breaklines among link reference that are breaking target format version. Could you help to merge this PR, please?
Many thanks!

cychua committed:
Add ref to SSSO

cychua committed:
Minor grammar fix

tfitzmac committed:
revising for updates

ErikjeMS committed:
example fix

tfitzmac committed:
fixing formatting issues

tfitzmac committed:
fix typo

iainfoulds committed:
Incorporating review feedback

mahesh-unnikrishnan committed:
Update active-directory-ds-join-windows-vm-template.md

christiankuhtz committed:
tweaks

itechedit committed:
Update how-to-read-write-files.md

JasonWHowell committed:
Update howto-configure-server-logs-in-cli.md

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into csharp-csharp-FW-update-article
sync with master before merge

christiankuhtz committed:
tweaks

ShawnJackson committed:
Merge pull request #25063 from itechedit/how-to-read-write-files

edit pass: how-to-read-write-files

Commit:
Merge pull request #25857 from JasonWHowell/patch-1

Update howto-configure-server-logs-in-cli.md

ShawnJackson committed:
Update site-recovery-replicate-vmware-to-azure.md

Commit:
Merge pull request #25849 from ErikjeMS/master

example fix

JasonWHowell committed:
MySQL: adding how-to configure server logs in portal

CamSoper committed:
added devcenter ms.custom tag

JasonWHowell committed:
MySQL: adding how-to configure server logs in portal

ktoliver committed:
Merge pull request #25834 from wmgries/release-notes-fix

Remove Azure File Sync service updates from Azure File Sync agent release notes page

PRMerger20 committed:
Merge pull request #25855 from christiankuhtz/patch-27

tweaks

paulathurman committed:
edit pass: data-prep-python-article

PRMerger14 committed:
Merge pull request #25862 from CamSoper/master

added devcenter ms.custom tag

JasonWHowell committed:
PostgreSQL: adding how-to configure server logs in portal

garyericson committed:
Fixed link

JasonWHowell committed:
Fixing links

ShawnJackson committed:
Merge pull request #25435 from paulathurman/data-prep-python-execution-api

edit pass: Data prep python execution api

GitHubber17 committed:
Merge pull request #25797 from mahesh-unnikrishnan/patch-15

Update instructions to deploy template to Azure from github.

GitHubber17 committed:
Merge pull request #25802 from mahesh-unnikrishnan/patch-17

Delete active-directory-ds-admin-guide-join-windows-vm-classic-powers…

sethmanheim committed:
Fixing image file names

PRMerger10 committed:
Merge pull request #25296 from ChrisGMsft/patch-2

Details on max/min values and sizes for twins

GitHubber17 committed:
Merge pull request #25847 from tfitzmac/1019market

revising for updates

GitHubber17 committed:
Merge pull request #24970 from JimacoMS2/csharp-csharp-FW-update-article

Add csharp-csharp firmware update article

PRMerger14 committed:
Merge pull request #25649 from MarkusVi/riskevents01

riskevents01

PRMerger10 committed:
Merge pull request #25842 from carolinacmoravia/patch-410

C5338: Broken format link in target versions due to missing breaklines

JasonWHowell committed:
Adding Alert article for PostgreSQL + MySQL

JasonWHowell committed:
Adding Alert article for PostgreSQL + MySQL

JasonWHowell committed:
Adding Alert article for PostgreSQL + MySQL

Commit:
Merge pull request #25762 from kfile/10182017TimeSeriesInsights

Updated date as part of a test to learn GitHub

vhorne committed:
change Access Control Policy

Commit:
Removed link to other multiple NIC article.

The other article was deprecated and redirects to this one now.

Commit:
Merge pull request #25877 from vhorne/upd-id

change Access Control Policy

HeidiSteen committed:
removing reference to score for KPE

HeidiSteen committed:
typo

ktoliver committed:
Merge pull request #25819 from sethmanheim/eh-metrics

Adding article on Event Hubs monitoring

itechedit committed:
Merge pull request #25860 from JasonWHowell/OrcasServerLogs

MySQL: adding how-to configure server logs in portal

PRMerger18 committed:
Merge pull request #25881 from cychua/master

Add SSSO ref

Commit:
Added link to Linux routing instructions.

Commit:
Merge pull request #25878 from jimdial/patch-7

Removed link to other multiple NIC article.

PRMerger10 committed:
Merge pull request #25884 from jimdial/patch-8

Added link to Linux routing instructions.

PRMerger19 committed:
Merge pull request #25880 from HeidiSteen/heidist-ta

removing reference to score for KPE

Lisaco88 committed:
Merge pull request #25467 from iainfoulds/vmssautoscale

VMSS auto scale updates

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

ShawnJackson committed:
Merge pull request #24530 from Lisaco88/site-recovery-replicate-vmware-to-azure

edit pass: site-recovery-replicate-vmware-to-azure

tynevi committed:
Merge pull request #25867 from garyericson/10-19-fix-links

Fixed link

Lisaco88 committed:
Merge pull request #25873 from JasonWHowell/OrcasAlerts

MySQL + PostgreSQL - adding How-To Monitor using Alerts

v-alje committed:
Merge pull request #25888 from MicrosoftDocs/FromPublicRepo

From public repo
